Kelvin_Lau_Project1.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kelvin_Lau_Project1</title>
+    <link href="../CSS/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <img id="aloha" src="../Assets/images/aloha-logo.svg" >
+      <nav>
+        <ul>
+          <li>ABOUT</li>
```

Comment:
Leave the ALL CAPS to the CSS with the text transform property. The markup should not have ALL CAPS.